grails-app/views/replenishment/_printPage.gsp

```diff
@@ -0,0 +1,103 @@
+<%@ page import="org.apache.commons.lang.StringEscapeUtils" defaultCodec="html" %>
+
+<div class="page" style="page-break-after: ${pageBreakAfter};">
+    <table id="requisition-items" class="fs-repeat-header" border="0">
+        <thead style="display: table-row-group">
+            <tr class="">
+                <td colspan="11">
+                    <h4 class="title">${groupName}</h4>
+                </td>
+            </tr>
+            <tr class="theader">
+                <th><warehouse:message code="report.number.label"/></th>
+                <th>${warehouse.message(code: 'product.productCode.label')}</th>
+                <th>${warehouse.message(code: 'product.name.label')}</th>
+                <th class="center" style="min-width: 150px;">${warehouse.message(code: 'default.lotSerialNo.label')}</th>
+                <th class="center">${warehouse.message(code: 'orderItem.expiry.label')}</th>
+                <th class="center">${warehouse.message(code: 'orderItem.transferToBin.label')}</th>
+                <th class="center">${warehouse.message(code: 'orderItem.qtyToTransfer.label')}</th>
+                <th class="center">${warehouse.message(code: 'inventoryLevel.binLocation.label')}</th>
+                <th class="center">${warehouse.message(code: 'requisitionItem.suggestedPick.label')}</th>
+                <th class="center" style="min-width: 100px">${warehouse.message(code:'default.notes.label')}</th>
+            </tr>
+        </thead>
+        <tbody>
+            <g:unless test="${lineItems}">
+                <tr>
+                    <td colspan="10" class="middle center">
+                        <span class="fade">
+                            <warehouse:message code="default.none.label"/>
+                        </span>
+                    </td>
+                </tr>
+            </g:unless>
+            <g:each in="${lineItems}" status="i" var="lineItem">
```

Comment:
Small improvement, but please change it to have a convention `each item in list` instead `each in list item` for better readability